worker/src/main/scala/Block.scala

```diff
@@ -0,0 +1,55 @@
+package worker
+
+import scala.io.Source
+import scala.util.Random
+
+import common._
+
+
+class Block(filepath: String){
+    val filename: String = filepath.split("/").last
+    val dir: String = filepath.split("/").dropRight(1).mkString("/")
+
+    var temp_dir: String = filepath
+    var key_range: (Int, Int) = (0, 0)
+    var file_size: Int = 0
+
+    var num_tuples: Int = 0
+
+    var buf: Array[Tuple] = Array()
+
+    def toStream: Stream[Tuple] = {
+        val source = Source.fromFile(filepath, "ISO8859-1")
+
+        def stream: Stream[Tuple] = {
+            if(!source.hasNext) Stream.empty
+            else {
+                val char_list = source.take(100).toList
+                val byte_list = char_list.map {_.toByte}
+                Tuple.fromBytes(byte_list) #:: stream
```

Comment:
```suggestion
                val byte_list = source.take(100).toList.map{_.toByte}
                Tuple.fromBytes(byte_list) #:: stream
```
`char_list`가 없어도 될 것 같네요